### PR TITLE
[Kotlin Migration] Counts 

### PR DESCRIPTION
**Pull request by codingtosh**
[Kotlin Migration] Counts 
package com.ichi2.libanki.sched

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/Counts.kt [hunk 1]
```diff
@@ -14,111 +14,69 @@
...
-                throw new RuntimeException("Index " + index + " does not exists.");
+     * @param number How much to add.
+     */
+    fun changeCount(index: Queue, number: Int) {
+        when (index) {
+            @Suppress("Redundant")
+            Queue.NEW -> new += number
+            Queue.LRN -> lrn += number
+            Queue.REV -> rev += number
+            else -> throw RuntimeException("Index $index does not exist.")
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/sched/Counts.kt:40]: Cleanup: I think we can remove the `else` branch in Kotlin

**Review by david-allison-1 — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/Counts.kt (hunk 1 above)
Comment (codingtosh) [reply AnkiDroid/src/main/java/com/ichi2/libanki/sched/Counts.kt:40]: asd

Comment (codingtosh) [reply AnkiDroid/src/main/java/com/ichi2/libanki/sched/Counts.kt:40]: getting this warning: 
`'when' expression on enum is recommended to be exhaustive, add 'NEW' branch or 'else' branch instead`

Comment (david-allison-1) [reply AnkiDroid/src/main/java/com/ichi2/libanki/sched/Counts.kt:40]: Looks like we handle `NEW`, but not worth questioning further

**Review by david-allison-1 — APPROVED** [review]
LGTM, ignoring one nit

**Review by mikehardy — APPROVED** [review]

**Pull request merged**